pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSinks.java

```diff
@@ -179,6 +199,10 @@ void runCmd() throws Exception {
         protected String namespace;
         @Parameter(names = "--name", description = "The sink's name")
         protected String name;
+
+        @Parameter(names = { "-t", "--sink-type" }, description = "The sinks's connector provider")
+        protected String sinkType;
```

Comment:
as a user, how can i know list of possible values?

Reply:
In #2106 I've added CLI commands to get the list, with the description help text:

```
pulsar-admin source available-sources
pulsar-admin sink available-sinks
```

Also, if you pass a wrong type, the error message will print out the valid choices.